### DISCUSSION TITLE
Add Norminette for CLion

### DISCUSSION THREAD

**Discussion by skrtks** [leeoocca/awesome-42#5] [New link]
Hey! I’ve created a Norminette plugin for CLion. Maybe you want to add it to your list?

 [You can find it here!](https://plugins.jetbrains.com/plugin/17190-norminette)

**Comment by leeoocca** [comment]
Sorry for the wait... 😅
I was away from keyboard enjoying the good season 🌞

That's wonderful!
Don't hesitate requesting a PR, respecting the [Formatting Guidelines](https://github.com/leeoocca/awesome-42/blob/master/CONTRIBUTING.md#formatting-guidelines) I outilined in [CONTRIBUTING.md](https://github.com/leeoocca/awesome-42/blob/master/CONTRIBUTING.md).

#### Edit
Oh, I was almost forgetting. I try to encourage people to look at source code when listing this kind of projects. So if you could add another bullet, like you see many projects already do, I would be perfect.

Something like this would be enough:
```
- [source code](https://github.com/skrtks/norminette)
```

Thanks!

**Comment by skrtks** [comment]
Cool, I've created a [request](https://github.com/leeoocca/awesome-42/pull/6)
